docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -0,0 +1,1164 @@
+
+Please note that that custom auth backends will need re-writing to target new FAB based UI.
+
+As part of this change, a few configuration items in ``[webserver]`` section are removed and no longer applicable,
+including ``authenticate``, ``filter_by_owner``, ``owner_mode``, and ``rbac``.
+
+Before upgrading to this release, we recommend activating the new FAB RBAC UI. For that, you should set
+the ``rbac`` options  in ``[webserver]`` in the ``airflow.cfg`` file to ``true``
+
+.. code-block:: bash
+
+    [webserver]
+    rbac = true
+
+In order to login to the interface, you need to create an administrator account.
+
+.. code-block:: bash
+
+    airflow create_user \
+        --role Admin \
+        --username admin \
+        --firstname FIRST_NAME \
+        --lastname LAST_NAME \
+        --email EMAIL@example.org
+
+If you have already installed Airflow 2.0, you can create a user with the command ``airflow users create``.
+You don't need to make changes to the configuration file as the FAB RBAC UI is
+the only supported UI.
+
+.. code-block:: bash
+
+    airflow users create \
+        --role Admin \
+        --username admin \
+        --firstname FIRST_NAME \
+        --lastname LAST_NAME \
+        --email EMAIL@example.org
+
+**Breaking Change in OAuth**
+
+The flask-ouathlib has been replaced with authlib because flask-outhlib has
+been deprecated in favour of authlib.
+The Old and New provider configuration keys that have changed are as follows
+
+==================== ==================
+Old Keys             New keys
+==================== ==================
+consumer_key         client_id
+consumer_secret      client_secret
+base_url             api_base_url
+request_token_params client_kwargs
+==================== ==================
+
+For more information, visit https://flask-appbuilder.readthedocs.io/en/latest/security.html#authentication-oauth
+
+
+Step 6: Upgrade Configuration settings
+'''''''''''''''''''''''''''''''''''''''''''
+
+Airflow 2.0 is stricter with respect to expectations on configuration data and requires explicit
+specifications of configuration values in more cases rather than defaulting to a generic value.
+
+Some of these are detailed in the Upgrade Check guide, but a significant area of change is with
+respect to the Kubernetes Executor. This is called out below for users of the Kubernetes Executor.
+
+**Upgrade KubernetesExecutor settings**
+
+*The KubernetesExecutor Will No Longer Read from the airflow.cfg for Base Pod Configurations.*
+
+In Airflow 2.0, the KubernetesExecutor will require a base pod template written in yaml. This file can exist
+anywhere on the host machine and will be linked using the ``pod_template_file`` configuration in the airflow.cfg.
+
+The ``airflow.cfg`` will still accept values for the ``worker_container_repository``, the ``worker_container_tag``, and
+the default namespace.
+
+The following ``airflow.cfg`` values will be deprecated:
+
+.. code-block:: bash
```

Comment:
```suggestion
The following ``airflow.cfg`` values will be deprecated::
```

This one is not bash -- and no special syntax is appropriate here.

Reply:
(Ending a block with `::` makes the next block a code block iff it is indented.)